torch_xla/csrc/lowering_context.cpp

```diff
@@ -263,8 +312,66 @@ void LoweringContext::AddParameter(const torch::lazy::Output& output,
   return;
 }
 
+// TODO: Remove duplicate code copied form openxla/xla MHLO -> HLO
+int64_t LoweringContext::AddStackFrameLocation(
```

Comment:
Can you put a link to openxla/xla code that has the similar code?

Reply:
Yes